app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
```

Comment:
Ok I've thought more about this now. Here is a dump of my thoughts...

My concern with the phrase "resources have descriptors" is that it seems like each resource could have its own set of descriptors; however, in reality, there is a global set of descriptors that every resource "responds" to. So the correct phrase is something more like "resources have descriptor_responses" or something. How does that sound? 

To be clear, the `descriptor` blueprint will contain functionality for changing the descriptors themselves, instead of changing individual resources' descriptor_responses? 

(I'm not proposing an immediate code change; I'm interested in starting a dialogue about what we're naming things.)